### PR TITLE
DFPL-1860 Applications to be sealed when uploaded

### PR DESCRIPTION
**Pull request by chak-shing-lo-justice**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-1860

### Change description ###
DFPL-1860 Applications to be sealed when uploaded

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: charts/fpl-case-service/values.preview.template.yaml [hunk 1]
```diff
@@ -155,6 +155,7 @@ ccd:
   ccd-api-gateway-web:
     nodejs:
       disableKeyVaults: true
+      image: hmctspublic.azurecr.io/ccd/api-gateway-web:prod-66e26b9-20231116094420
```
Comment (chak-shing-lo-justice) [inline charts/fpl-case-service/values.preview.template.yaml:158]: Will revert this changes once ccd-api-gw pod stop crashing

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/additionalapplications/UploadAdditionalApplicationsServiceTest.java [hunk 2]
```diff
@@ -457,43 +468,48 @@ void shouldNotSkipPaymentsWhenApplyingForAnOtherOrder() {
...
 
+            assertThat(converted.getDocument())
+                .isEqualTo(SEALED_DOCUMENT);
             assertThat(converted.getSupplementsBundle().get(0).getValue().getDocument())
                 .isEqualTo(CONVERTED_SUPPLEMENT_DOCUMENT);
         }
 
         @Test
-        void shouldConvertOtherSupplementToPdf() {
+        void shouldSealOtherDocumentIf() {
```
Comment (jamesrferguson1) [inline service/src/test/java/uk/gov/hmcts/reform/fpl/service/additionalapplications/UploadAdditionalApplicationsServiceTest.java:502]: maybe check the name here ... if what? :)

Comment (chak-shing-lo-justice) [reply service/src/test/java/uk/gov/hmcts/reform/fpl/service/additionalapplications/UploadAdditionalApplicationsServiceTest.java]: fixed to shouldSealOtherDocument

**Review by so99y — APPROVED** [review]
